package exception;

public class ExceptionDemo {
	
	public static void main(String[] args) {
		
		int a = 3;
		int b = 0;
		
		try {
			int c = a/b;
		}catch(ArithmeticException e) {
			
		}
		
		
		System.out.println(c);
		
		int d = a*b;
		
		System.out.println(d);
	}

}
